src/instruments/src/EFB/TODCalculator/Data/Data.tsx

```diff
@@ -0,0 +1,78 @@
+/*
+ * A32NX
+ * Copyright (C) 2020-2021 FlyByWire Simulations and its contributors
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { connect } from 'react-redux';
+import Input from "../../Components/Form/Input/Input";
+import React from "react";
+import Card from "../../Components/Card/Card";
+import Divider from "../../Components/Divider/Divider";
+import {TOD_CALCULATOR_REDUCER} from "../../Store";
+import {setTodData} from "../../Store/action-creator/tod-calculator";
+import {TOD_CALCULATION_TYPE} from "../../Enum/TODCalculationType.enum";
+
+function Data({
```

Comment:
Same here

Reply:
done

Reply:
i think we can add this rule https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/prefer-stateless-function.md in some of future changes